### ISSUE TITLE
[Suggestion] Automatically convert .ipynb and .Rmd files in the user's repo to Dar

### ISSUE THREAD

**Issue by nokome** [minrk/jupyter-dar#8]
There are converters in https://github.com/stencila/convert which allow for conversion to/from various formats including Dar. For Articles (i.e. text documents) it uses Pandoc 2.1.3 (which has both a JATS reader and writer in it and happens to be the version of Pandoc in the Docker image already!)

```bash
npm install stencila/convert -g
cd dir
stencila-convert import .
```

where `dir` is the directory where the `.ipynb` or `.Rmd` file resides.

**Comment by nuest** [comment]
Not sure I get this. _How could a user trigger that conversion?_ Unless there is a Dar article in the repo, the Binder will not have Stencila available... once could "force" having Stencila by adding a Dockerfile to the repo, but then I can also do the conversion before starting the Binder..

**Comment by nokome** [comment]
Yes, I was imagining that a conversion would get triggered when the user visited the `/stencila` endpoint to create a Dar. We're doing something like that over in [`stencila/hub`](https://github.com/stencila/hub/blob/9267e5e9665ed46e01f4d7854d44996cc0ae4b18/director/director/models.py#L120). 

It's not trivial though and may be it's a bit out of scope for this project (I was getting excitedly carried away :) I'll close this issue.
